Description: Upstream changes introduced in version 2010.05.03-1ubuntu4
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 openscadpy (2010.05.03-1ubuntu4) natty; urgency=low
 .
   * .desktop added
 .
 The person named in the Author field signed this changelog entry.
Author: Henning Meyer <hmeyer@gmx.eu>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- /dev/null
+++ openscadpy-2010.05.03/openscadpy.desktop
@@ -0,0 +1,7 @@
+[Desktop Entry]
+Type=Application
+Version=1.0
+Name=OpenSCADpy
+Icon=openscadpy
+Exec=openscadpy %f
+Categories=Graphics;3DGraphics;Engineering;Python
